plugin-api.d.ts

```diff
@@ -200,6 +208,29 @@ interface RunEvent<ParametersType = (ParameterValues | undefined)> {
   parameters: ParametersType
 }
 
+interface DropEvent {
+  node: BaseNode | SceneNode
+  x: number
+  y: number
+  absoluteX: number
+  absoluteY: number
+  items: DropItem[]
+  files: DropFile[]
+  dropMetadata?: any
+}
+
+interface DropItem {
+  type: string
+  data: string
+}
+
+interface DropFile {
```

Comment:
I think you suggested `SimpleFile` somewhere which I like a lot. I was thinking FigmaFile, but that sounds too much like a Figma document, or PluginFile, but that sounds like a plugin source resource. `APIFile` could be another option.

Also keeping `DropFile` is totally fine. We can use it elsewhere too even though the name would be awkward. Worst case we can change the name later since that won't break existing plugins.

Reply:
👍 I added a task to the Plugins project in Asana so that we can remember to rename this in the future when we need to reuse this interface elsewhere.